<launch>
    <rosparam command="load" file="$(find my_pcl)/config/point_topic.yaml"/>
    <!-- Launch front_cam -->
    <include file="$(find realsense2_camera)/launch/front_cam.launch">
    </include>
    <!-- Launch back_cam -->
    <include file="$(find realsense2_camera)/launch/back_cam.launch">
    </include>
    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="back_to_front" args="-0.7071 0.0 -0.7071 3.1416 1.5708 0.0 back_cam_link front_cam_link" />  -->

    <node pkg="my_pcl" type="pcl_write" name="pcl_write" output="screen" >
    </node>
</launch>
